### ISSUE TITLE
Module parse failed: Bad character escape sequence

### ISSUE THREAD

**Issue by prakhardubey2002** [vercel/next.js#71625]
### Link to the code that reproduces this issue

https://github.com/prakhardubey2002/test

### To Reproduce

>> npx create-next-app@latest
>> npx shadcn@latest init
>> npx shadcn@latest add "https://magicui.design/r/marquee"
>> npm run dev

### Current vs. Expected behavior

The app is not starting in local environment even though it's just fresh boiler plate of next js with magic UI component
![image](https://github.com/user-attachments/assets/0bf0f137-6acc-4cf8-bf32-4a4954ea6528)

### Provide environment information

```bash
Operating System:
  Platform: win32
  Arch: x64
  Version: Windows 11 Home Single Language
  Available memory (MB): 7927
  Available CPU cores: 8
Binaries:
  Node: 20.10.0
  npm: N/A
  Yarn: N/A
  pnpm: N/A
Relevant Packages:
  next: 15.0.0 // Latest available version is detected (15.0.0).
  eslint-config-next: 15.0.0
  react: 19.0.0-rc-65a56d0e-20241020
  react-dom: 19.0.0-rc-65a56d0e-20241020
  typescript: 5.6.3
Next.js Config:
  output: N/A
```

### Which area(s) are affected? (Select all that apply)

Not sure

### Which stage(s) are affected? (Select all that apply)

next dev (local), next build (local), next start (local)

**Comment by lubieowoce** [comment]
Duplicate of #71582

**Comment by devjiwonchoi** [comment]
Hey @prakhardubey2002, sorry for letting this slip. Feel free to ping me if [15.0.1-canary.2](https://github.com/vercel/next.js/releases/tag/v15.0.1-canary.2) did not fix it. When we can confirm, will have the patch ASAP to unblock your upgrade. Again, apologies for inconvenience and thank you for your report. Thank you!

**Comment by Muzi12345632** [comment]
Bug is still there

**Comment by devjiwonchoi** [comment]
@Muzi12345632 Lmk how it goes with [v15.0.2-canary.10](https://github.com/vercel/next.js/releases/tag/v15.0.2-canary.10)

**Comment by devjiwonchoi** [comment]
v15.0.2 is out with the fix. Please give it a try and lmk!
